modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/GlobalCSSCETImplFactoryImpl.java

```diff
@@ -62,7 +62,9 @@ public void validate(
 
 		if (!Validator.isUrl(newGlobalCSSCET.getURL())) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"please-enter-a-valid-url");
+				"Please enter a valid URL",
+				"please-enter-a-valid-url"
```

Comment:
This isn't a part of your change, but I think we should take this opportunity to add consistency here: this should match the exception above: `"url-x-is-invalid", url`